paper/implementation.tex

```diff
@@ -62,33 +62,7 @@ \subsection{Tool Overview}\label{subsec:tool-overview} %keyuri
 
 \subsubsection{Toolbar functionalities}
 
-The toolbar provides basic functionalities for all three views:
```

Comment:
I had compile errors in this section somewhere and Benjamin said we should remove it anyway